distribution/scripts/start-client.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+
+# Wrapper to force the client to wait for the backend to become available
+# Should only be useful during development
+set -euo pipefail
+
+sleepTime=8
+if [ ! -z "${DEVELOPMENT:-}" ]; then
+  echo "DEVELOPMENT MODE: client will wait $sleepTime before starting, to give time to the backend to start and receive a first UL"
+  sleep $sleepTime
+else
+  echo "Client is starting up"
+fi
+
+maxAttempts=30
+
+until curl -s "$EVERGREEN_ENDPOINT" --output /dev/null ; do
```

Comment:
Is this expected to be running in non-development mode as well? In released instances, the client will exit if it cannot reach the backend and allow supervisord to restart it.

Is this change of behavior intentional for those released instances? If so, would you mind adding some comments to the script as to why this is necessary.

Reply:
There are already a pretty commit log with this but ok

Reply:
OK, I think I get your point now. Yes, I could, probably should, move that block inside the `if` above too. Will do.

Reply:
fixed with latest pushed commit.